test/xml-generator/Class.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SystemClass Name="Test">
+    <SubsystemInclude Path="Subsystem.xml"/>
+</SystemClass>
```

Comment:
Are you sure this file is rely needed ? Is it adding something in your coverage ?